src/compiler/transformers/ts/CommonTSfeaturesTransformer.ts

```diff
@@ -34,29 +35,33 @@ export function CommonTSfeaturesTransformer(): ITransformer {
             return { removeNode: true, ignoreChildren: true };
           }
 
-          if (node.type === 'TypeAssertion' || node.type === 'NonNullExpression') {
+          if (node.type === ASTType.TypeAssertion || node.type === ASTType.NonNullExpression) {
             return { replaceWith: node.expression };
           }
-
-          if (node.type === 'MethodDefinition' && node.value && node.value.type === 'EmptyBodyFunctionExpression') {
+          // EmptyBodyFunctionExpression is a specific buntis key
```

Comment:
is in buntis the body also empty?

if so...
```
if (
  node.type === 'MethodDefinition' &&
  node.value &&
  !node.value.body
) {
```

Reply:
let's be explicit here ;-)